api/types/filters/parse.go

```diff
@@ -211,20 +210,10 @@ func (args Args) GetBoolOrDefault(key string, defaultValue bool) (bool, error) {
 
 	isFalse := fieldValues["0"] || fieldValues["false"]
 	isTrue := fieldValues["1"] || fieldValues["true"]
-
-	conflicting := isFalse && isTrue
-	invalid := !isFalse && !isTrue
-
-	if conflicting || invalid {
+	if isFalse == isTrue {
```

Comment:
Not sure that's helpful 🙈 

Maybe something like:
```suggestion
    // Either no or conflicting truthy/falsy value were provided 
	if isFalse == isTrue {
```